app/build.gradle

```diff
@@ -95,12 +96,18 @@ android {
             signingConfig = signingConfigs.debug
 
             resValue "string", "content_provider", "com.dhis2.debug.provider"
+            buildConfigField "String", "ACRA_USER", "\"vdMEmIrI9uJhidAt\""
+            buildConfigField "String", "ACRA_PASSWORD", "\"SRFVoyVMVMtTI5PY\""
+            buildConfigField "String", "MATOMO_URL", "\"https://mipruebacustom.matomo.cloud/matomo.php\""
         }
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             ext.enableCrashlytics = true
 
+            buildConfigField "String", "ACRA_USER", "\"vdMEmIrI9uJhidAt\""
+            buildConfigField "String", "ACRA_PASSWORD", "\"SRFVoyVMVMtTI5PY\""
+            buildConfigField "String", "MATOMO_URL", "\"https://mipruebacustom.matomo.cloud/matomo.php\""
```

Comment:
Acra credentials changes depending on build variant. Matomo URL change depending on build variant

Reply:
I don't think is a good idea to upload credentials into github. We should look into setting this as environmental variables in bitrise.

Reply:
Very good point @ferdyrod

Reply:
yep. was checking it out today. I'll use system.env .
We have to agree everyone to set this variables in our local machines too, so that crashes are reported.